samples/grpc_server/route_guide.proto

```diff
@@ -1,25 +1,12 @@
-// Copyright 2015 gRPC authors.
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
```

Comment:
Should we keep it now that we changed it?

Reply:
Yeah, I think we must keep it, but maybe also either do something like this:
```
Copyright 2015 gRPC authors, with some modification by the k6 team (2021).
```

or just leave it as it is :man_shrugging:

Reply:
@na-- sorry, I pushed before I was able to read your comment, I will add your suggestion in the second PR, we expect to create.